config/config.go

```diff
@@ -223,6 +239,127 @@ func (c *Config) Validate() error {
 	return nil
 }
 
+// If the numerator in the Warp config is 0, use the default value
+func setQuorumNumerator(cfgNumerator uint64) uint64 {
+	if cfgNumerator == 0 {
+		return params.WarpDefaultQuorumNumerator
+	}
+	return cfgNumerator
+}
+
+// Helper to retrieve the Warp Quorum from the chain config.
+// Differentiates between subnet-evm and coreth RPC internally
+func getWarpQuorum(
+	blockchainID ids.ID,
+	subnetID ids.ID,
+	client ethclient.Client,
+) (WarpQuorum, error) {
+	if subnetID == constants.PrimaryNetworkID {
+		return WarpQuorum{
+			QuorumNumerator:   params.WarpDefaultQuorumNumerator,
```

Comment:
why do we set to default quorum numerator for c-chain? Don't we still use it when aggregating signature

Reply:
For a Warp message originating on the c-chain, when we aggregate signatures do we use the quorum numerator of the destination chain? Right now in `message_relayer` it looks like we still use the quorum numerator of the c-chain

Reply:
> For a Warp message originating on the c-chain, when we aggregate signatures do we use the quorum numerator of the destination chain

Yes, we should always be using the destination's warp quorum to construct the aggregate signature. I think there were some commits here and in the latest PR https://github.com/ava-labs/awm-relayer/pull/157 in which that wasn't the case. Please let me know if you still see an issue.